### DISCUSSION TITLE
Load Trained Model on Carla Simulator

### DISCUSSION THREAD

**Discussion by Dilcan44** [carla-simulator/carla#7695] [Q&A]
Hi everyone, 
Im training a model with stable_baselines3 and it actually works fine, but once I stop the training i can't load the model, it just starts from 0 again, does anybody know how i can do it?
Thanks in advance

**Comment by xavisolesoft** [comment]
Could you share the code/commands you are running to reproduce this issue, please?

**Reply by Dilcan44** [reply]
Estoy utilizando este codigo que proporcionaba otra persona de la comunidad, he probado alguna otra versión pero siempre me carga una versión del modelo sin entrenar. 

![Codi](https://github.com/carla-simulator/carla/assets/150742352/a8fca7a3-ea70-41e8-ada1-6f06fd9d26d3)
